### PR TITLE
Refactor loadgen to handle IST

### PR DESCRIPTION
**Pull request by mhofman**
https://github.com/Agoric/agoric-sdk/issues/5326 is renaming RUN -> IST. This PR refactors the loadgen to no longer hardcode `RUN`, and instead names anything related with "stable". Then it adds a dynamic fallback from IST to RUN for backwards compatibility.

Successfully tested locally the "local-chain" variant against the same revisions as https://github.com/Agoric/testnet-load-generator/pull/71, as well as the current master of agoric-sdk: `12e61941`. The "testnet" variant against the same agoric-sdk master revision is also passing, allowing a safe merge without breaking agoric-sdk's CI.

Tested against the [IST rename branch](https://github.com/Agoric/agoric-sdk/pull/5585) (`4800-unRUN`).

### CODE REVIEW

**Comment by mhofman** [comment]
Older revisions with Zoe fee purse are failing. I need to decide how ugly of a patch I want to handle that case.

File: loadgen/contract/agent-create-vault.js [hunk 1]
```diff
@@ -25,25 +25,26 @@ async function getPurseBalance(purse) {
  *
  * @param {startParam} param
  * @typedef {Awaited<ReturnType<typeof startAgent>>} Agent
- * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'name'>} AssetKit
+ * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'symbol'>} AssetKit
```
Comment (michaelfig) [inline loadgen/contract/agent-create-vault.js:28]: Technically, this is not a `symbol`, it's a Zoe `keyword`.  Unfortunately, this change has further repercussions below.

```suggestion
 * @typedef { Pick<import('../types').NatAssetKit, 'brand' | 'purse' | 'displayInfo' | 'keyword'>} AssetKit
```

**Review by michaelfig — CHANGES_REQUESTED** [review]
Looks good, but I'd strongly encourage you to use `keyword` instead of `symbol`.

**Comment by mhofman** [comment]
> Looks good, but I'd strongly encourage you to use `keyword` instead of `symbol`.

I went by the nomenclature @dckc seem to have adopted in his PR. I did however just switch to a "keyword" name for vaults. Currently the keyword for those is still `RUN` which is obviously wrong, so making this PR draft until that gets settled.

**Comment by mhofman** [comment]
Rebased PR on latest main, and update to use the `Minted` keyword for vaults according to https://github.com/Agoric/agoric-sdk/pull/5585. However that PR does not currently incorporate https://github.com/Agoric/agoric-sdk/pull/5332 which this was originally based against, so right now it'll not work (I've verified that this PR still runs correctly against the `5328-IST-boot`  branch if the vault keyword is forced to `RUN` in the vault task agent).

**Review by turadg — APPROVED** [review]
[Passed](https://github.com/Agoric/agoric-sdk/actions/runs/2742246120) against the last https://github.com/Agoric/agoric-sdk/pull/5585. Changes to that branch since then have been irrelevant tweaks (docs, types, methods that aren't used here.)

**Pull request merged**